DownFall/RuneDetailView.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  RuneDetailView.swift
+//  DownFall
+//
+//  Created by Katz, Billy on 4/19/20.
+//  Copyright © 2020 William Katz LLC. All rights reserved.
+//
+
+import SpriteKit
+
+struct RuneDetailViewModel {
```

Comment:
We should really add tests for all these.  Define the interface in a protocol and test the interface

Reply:
Classic dev thing to say, but ill add tests when the feature is finished, I dont want to update tests if this changes a bunch